src/commands/install.yml

```diff
@@ -1,22 +1,26 @@
 description: >
-  Install MATLAB on a Linux machine executor. Currently, this command is available only for public
-  projects and does not include transformation products, such as MATLAB Coder and MATLAB Compiler.
+  Install a specified version of MATLAB. If you use this command to install transformation products, such
+  as MATLAB Coder and MATLAB Compiler, the command does not automatically license such products for you.
```

Comment:
The transformation product restriction gets a lot of attention here. Maybe highlighting a few more important points first would help?